### DISCUSSION TITLE
coc-java

### DISCUSSION THREAD

**Discussion by Eduardo-Guerra-Alvarez** [neoclide/coc.nvim#4614] [Extension request]
I want to install coc-java extension but show a message "coc-java 1.14.1 requires coc.nvim >= >=0.0.82, please update coc.nvim."
How can I update coc to 0.0.82, I tried with PlugUpdate but this only update with 0.0.81

**Comment by jackielii** [comment]
using master should just work:

```
Plug 'neoclide/coc.nvim', {'branch': 'master', 'do': 'yarn install --frozen-lockfile'}
```

**Comment by fannheyward** [comment]
> How can I update coc to 0.0.82

Depends on how do you install coc.nvim. Check your vim plugin manager and your configurations.
